### ISSUE TITLE
Page Load disaster - TTFB of 4 seconds after upgrading 2.2.3 > 2.2.4

### ISSUE THREAD

**Issue by johnny-longneck** [magento/magento2#15474]
After upgrading Magento 2.2.3 to 2.2.4 I have now a TTFB of around 4 seconds.
Every page I load has a delay of these 4 seconds now. Before it was immediatly there. Being often complimented about "how fast the side is".
Now it's a disaster.
Varnish is enabled and Nginx SSL offloading. 

Only in live envionment. Attached a screenshot with the profiler result. 
Please please help.

### Preconditions
PHP 7.0.30
Varnish 4 listens on Port 80, Nginx on 8080 and 443. SSL Offloading. (Using the varnish template since 2.1. - the new exported one does not work and causes everything to break)
Magento 2.2.4
Mysql

### Steps to reproduce

1. Occurs on every page. Only in production mode on live server, not locally.
Not sure what the problem really is. 

### Expected result

1. Page loads as expected fast

### Actual result

1. TTFB is always 4 seconds now causing a big delay on the page load

![ttfb-wtf](https://user-images.githubusercontent.com/25068173/40467336-cb39a61e-5f29-11e8-84e9-c59c9626badd.png)

**Comment by johnny-longneck** [comment]
The problem already occurs when doing php bin/magento

```
time php -v : real	0m0.170suser	0m0.056ssys	0m0.068s
time php bin/magento real	0m27.561suser	0m26.364ssys	0m1.000s
```
Local machine - same code - everything works
Remote machine - the problem comes up. 

I am super frustrated. Any help highly appreciated

**Comment by DanielRuf** [comment]
Can you easily create a clone to test with `git bisect`?

**Comment by johnny-longneck** [comment]
I cloned everything to a new server with the same result. Trying to disable and uninstall each module one by one. 

I think git bisect is not an option here. 

I traced the problem (in a really bad way, don't ask how) down so that for now I can say
php bin/magento gets stuck when creating the $application object on file
vendor/magento/framework/Console/Cli.php
in construct method line

`$this->initObjectManager()`

I think I have read something about a generated permissions bug. I will check this.
